src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -261,6 +261,9 @@ javac.opt.Xlint.desc.varargs=\
 javac.opt.Xlint.desc.preview=\
     Warn about use of preview language features
```

Comment:
Old code, I know, but the convention appears to be that these descriptions should end in a period. (Same comment for both `javac.opt.Xlint.desc.varargs` and `javac.opt.Xlint.desc.preview`.)

Reply:
OK, will do.